reports/Google-2018-report.md

```diff
@@ -13,13 +15,18 @@ prioritize transparency, diversity, and reproducibility to foster collaboration.
 Imagine you are a carrot farmer trying to make dinner. You could make carrot soup by yourself, but you could also share ingredients 
 with the neighbours to make a much tastier soup with onions, greens, and some salt. Some people may be able to contribute more than others -- but everyone benefits.
 
+In neuroscience, data such as magnetic resonance images and physiology readings can be costly in both time and money to acquire. This creates a barrier for many underfunded researchers without access to the required equipment. Responsible data sharing can level the playing field, but incompatible specifications cause portability issues between different labs and scientists. The brain imaging data standard (BIDS) is a framework for organizing data that standardizes datasets across different researchers to enable easier sharing.
 
 
-In neuroscience, data such as magnetic resonance images and physiology readings can be costly in both time and money to acquire. This creates a barrier for many underfunded researchers without access to the required equipment. Responsible data sharing can level the playing field, but incompatible specifications cause portability issues between different labs and scientists. The brain imaging data standard (BIDS) is a framework for organizing data that standardizes datasets across different researchers to enable easier sharing.
+## My role
 
-## Our role
+The primary goal of this project was to simplify the process of learning about BIDS and engage more users.
+Well before it was funded by Google, the roots of this initiative were being pioneered by a small group of people around the world that were acting as local BIDS ambassadors in their communities.
+Many of us were already giving presentations at every workshop and conference possible.
+However, these were only for a limited audience, and there was a need for central organization.
```

Comment:
"...organization of these resources." Here, just "organization" reads unclear to me (i.e., do you mean a BIDS inc?)

Reply:
:+1: Incorporated :)